Arduboy.cpp

```diff
@@ -682,14 +687,12 @@ unsigned char* Arduboy::getBuffer()
 
 boolean Arduboy::pressed(uint8_t buttons)
 {
- uint8_t button_state = getInput();
- return (button_state & buttons) == buttons;
```

Comment:
>What about pollButtonState()?

At this point, I think *getInput()* has been used too heavily to justify changing it. I don't think *getInput()* is too confusing, as long as it's well documented (Doxygen makes that easy, doesn't it?)